.storybook/main.ts

```diff
@@ -15,5 +17,15 @@ const config: StorybookConfig = {
   docs: {
     autodocs: "tag",
   },
+  webpackFinal: async (config) => {
+    config.resolve = config.resolve || {};
+    config.resolve.alias = config.resolve.alias || {};
+    config.resolve.alias["@"] = path.resolve(__dirname, "../src");
+
+    config.plugins = config.plugins || [];
```

Comment:
```suggestion
    config.resolve = config.resolve ?? {};
    config.resolve.alias = config.resolve.alias ?? {};
    config.resolve.alias["@"] = path.resolve(__dirname, "../src");

    config.plugins = config.plugins ?? [];
```
이런 부분도 Nullish Coalescing Operator가 습관화되면 좋을 것 같습니다!

Reply:
감사합니다 수정했습니다 !